Mathlib/Algebra/GroupWithZero/Pointwise/Finset.lean

```diff
@@ -190,7 +190,7 @@ open scoped RightActions
       MulOpposite.op_inv, inv_eq_zero, MulOpposite.op_eq_zero_iff, inv_inv,
       MulOpposite.smul_eq_mul_unop, MulOpposite.unop_op, mul_inv_rev, ha]
 
-@[simp] lemma inv_op_smul_finset_distrib₀ (a : α) (s : Finset α) : (s <• a)⁻¹ = a⁻¹ • s⁻¹ := by
+lemma inv_op_smul_finset_distrib₀ (a : α) (s : Finset α) : (s <• a)⁻¹ = a⁻¹ • s⁻¹ := by
```

Comment:
Can you (write a reply to this comment to) explain this change?

Reply:
Moving lemmas around made simp nf time out on this lemma. I have no idea why